### PR TITLE
Fix empty traces in X-Ray console

### PR DESCRIPTION
**Pull request by nachomdo**
👉  https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1708

### CODE REVIEW

File: exporter/awsxrayexporter/translator/segment.go [hunk 1]
```diff
@@ -80,10 +80,11 @@ func MakeSegmentDocumentString(span pdata.Span, resource pdata.Resource, indexed
 
 // MakeSegment converts an OpenTelemetry Span to an X-Ray Segment
 func MakeSegment(span pdata.Span, resource pdata.Resource, indexedAttrs []string, indexAllAttrs bool) (*awsxray.Segment, error) {
-	var segmentType string
+	var segmentType string = "segment"
```
Comment (anuraaga) [inline exporter/awsxrayexporter/translator/segment.go:83]:
```suggestion
	var segmentType string
```

We only need to set the type for subsegments

**Review by anuraaga — COMMENTED** [review]
Thanks! Can you add a unit test?

**Comment by nachomdo** [comment]
> Thanks! Can you add a unit test?

Thanks for the review @anuraaga. I have added a unit test to cover the case where we have non server span kind without parentID and left the default segmentType as it was.

**Review by anuraaga — APPROVED** [review]
Thanks!

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**